### ISSUE TITLE
Enhance alert of `cannotEncrypt` function

### ISSUE THREAD

**Issue by vulopez** [vulopez/text-scrambler#9]
# Description

Think in accessibility doing the error message more visible for the user. The #8 merge paint of red the message but should be more visible.
